vendor/assets/javascripts/dataTables/jquery.dataTables.js

```diff
@@ -1527,12 +1527,17 @@
   
     return out;
   };
-  
-  
-  var _stripHtml = function ( d ) {
-    return d
-      .replace( _re_html, '' ) // Complete tags
-      .replace(/<script/i, ''); // Safety for incomplete script tag
```

Comment:
How are we stripping <script tags ?

Reply:
I see what you talking about. Let me work on this

Reply:
I think this `d.replace(/<|>/g, "")` solution fix all tegs problem.

And specially for CodeQL 

We explicitly strip script tag `tempDiv.querySelectorAll('script');`